eslint src fails after cloning repo

**Describe the bug**
When running npm install just after running the repository, the last eslint src fails

**To Reproduce**
Steps to reproduce the behavior:
1. Clone the repo
2. run "npm install"
3. See the following error messages:
/Users/djazouli/WebstormProjects/test/react-digraph/src/components/graph-view.js
  475:7  error  'canSwapEdge' is not defined  cup/no-undef

/Users/djazouli/WebstormProjects/test/react-digraph/src/index.js
  28:9  error  Replace `⏎··default·as·BwdlTransformer,⏎` with `·default·as·BwdlTransformer·`  prettier/prettier

**Expected behavior**
eslint should not fail

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
OS: macOS Catalina v10.15.5
Browser: Brave
Version 1.10.97 Chromium: 83.0.4103.116 (Official Build) (64-bit)

**Additional context**
Add any other context about the problem here.

--- a/src/components/graph-view.js
+++ b/src/components/graph-view.js
@@ -467,6 +467,8 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
   }
 
   canSwap(sourceNode: INode, hoveredNode: INode | null, swapEdge: any) {
+    const { canSwapEdge } = this.props;
+
     return (
       hoveredNode &&
       sourceNode !== hoveredNode &&

--- a/src/index.js
+++ b/src/index.js
@@ -25,9 +25,7 @@ export type IEdgeType = IEdge;
 export { default as GraphUtils } from './utilities/graph-util';
 export { default as Node } from './components/node';
 export type INodeType = INode;
-export {
-  default as BwdlTransformer,
-} from './utilities/transformers/bwdl-transformer';
+export { default as BwdlTransformer } from './utilities/transformers/bwdl-transformer';
 export { GV as GraphView };
 export type LayoutEngineType = LayoutEngineConfigTypes;
 export default GV;

